contracts/mocks/lib/CommonMathMock.sol

```diff
@@ -24,6 +24,26 @@ contract CommonMathMock {
         return CommonMath.safePower(a, pow);
     }
 
+    function testScale(
```

Comment:
Question: What are your views for using this `scale` pattern to add additional precision when doing math?